VadimVildanov/HW7/Task3.py

```diff
@@ -0,0 +1,22 @@
+# I create function with argument "word". "word" is word which will enter user.
+def calculates_characters(word):
+    # I decided do with dict and save letters
+    dict = {}
```

Comment:
It's a good practice not to use the name dict for variables because it's a built-in type in Python. You can use a more descriptive name like character_count or something similar.